src/js/control-bar/time-controls/remaining-time-display.js

```diff
@@ -47,6 +47,7 @@ class RemainingTimeDisplay extends TimeDisplay {
    * @private
    */
   formatTime_(time) {
+    // TODO: The "-" should be decorative, and not announced by a screen reader
```

Comment:
this is because the screen reader would announce something like "remaining time minus 5:00" as it is currently?

Reply:
Exactly. The "-" is a visual convention, but the actual amount of time remaining is not negative. It's a small detail, which is why I didn't fix it now, but it ought to be fixed in the future.

Reply:
we should probably open an issue so we don't forget

Reply:
Done: see #5168.